.travis.yml

```diff
@@ -16,6 +16,7 @@ install:
 - yarn install
 script:
 - ./scripts/ci/travis-check.js
+- ./scripts/ci/trigger-ckeditor5-continuous-integration.sh
```

Comment:
I'm wondering which build should trigger the script. We have three options:

1. Tests/Coverage
2. Validation (Linters/Manual tests)
3. Both.

The last option (3) will duplicate the job because, for a single commit, we will execute the same action twice.

The Validation step requires slightly much more time for checking everything so that I will go with the (1) option. 

Also, the Validation step may wait in the queue if we will have 3 builds (for different commits, e.g., PRs) at the same time.